stl/inc/random

```diff
@@ -261,30 +261,37 @@ private:
     vector<result_type> _Myvec;
 };
 
-_NODISCARD constexpr int _Generate_canonical_iterations(const int _Bits, const uint64_t _Gmin, const uint64_t _Gmax) {
-    // For a URBG type `G` with range == `(G::max() - G::min()) + 1`, returns the number of calls to generate at least
-    // _Bits bits of entropy. Specifically, max(1, ceil(_Bits / log2(range))).
-
-    _STL_INTERNAL_CHECK(0 <= _Bits && _Bits <= 64);
+struct _Urbg_gen_can_params {
+    int _Kx          = 0; // $k$ in [rand.util.canonical]/1.4
+    uint64_t _Xx     = 0; // $x$ in [rand.util.canonical]/1.4

```

Comment:
```suggestion
    uint64_t _Xx     = 0; // $x$ in [rand.util.canonical]/1.5

```

Reply:
When citing a paragraph, it's important to include the WP number, because paragraphs are frequently renumbered. (Stable names by themselves are stable enough that they don't need WP numbers.)